.github/workflows/pre-commit-autoupdate.yml

```diff
@@ -0,0 +1,36 @@
+name: Autoupdate
+on:
+  schedule:
+    - cron: '0 0 1 * *'
+jobs:
+  pre-commit:
+    name: Pre-commit
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup git
+        run: |
+          git config --global user.name "Pre-commit automation"
+          git config --global user.email "releng+pre-commit-autoupdate@mozilla.com"
+      - uses: actions/setup-python@v5
+      - run: python -m pip install pre-commit
+        shell: bash
+      - name: Update files
+        run: |
+          git checkout -B pre-commit-update
+          pre-commit autoupdate -j4
+          git add .pre-commit-config.yaml
+          git commit -am "Update pre-commit"
```

Comment:
Would it be too annoying to parse:
https://github.com/taskcluster/taskgraph/blob/main/.pre-commit-config.yaml#L6

instead? Or at the very least remove that config (and I guess the "schedule" one too). Fwiw, I like the configured commit message better.

Reply:
Changed it to use `ci.autoupdate_commit_msg` if it exists and removed keys that were only used by the app

Reply:
I'll use the same for the PR title actually since we squash things